### PR TITLE
Updated A references

### PR DESCRIPTION
**Pull request by dannago**

### CODE REVIEW

File: sp800-63a/sec10_references.md [hunk 1]
```diff
@@ -6,27 +6,25 @@
...
 
 <a name="dmf"></a>[DMF] National Technical Information Service, *Social Security Death Master File*, available at: <https://www.ssdmf.com/Library/InfoManage/Guide.asp?FolderID=1>.
 
 <a name="E-Gov"></a>[E-Gov] *E-Government Act of 2002* \(includes FISMA) (P.L. 107-347), December 2002, available at: <http://www.gpo.gov/fdsys/pkg/PLAW-107publ347/pdf/PLAW-107publ347.pdf>.
 
-<a name="9397"></a>[EO 9397] *Executive Order 9397, Numbering System for Federal Accounts Relating to Individual Persons*, November 22, 1943, available at: <https://www.ssa.gov/foia/html/EO9397.htm>.
-
-<a name="fbcacp"></a>[FBCACP] X.509 Certificate Policy For The Federal Bridge Certification Authority (FBCA), Version 2.27, December 2, 2013, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TN7cAAG&field=File__Body__s>.
+<a name="fbcacp"></a>[FBCACP] X.509, *Certificate Policy For The Federal Bridge Certification Authority (FBCA)*, Version 2.27, December 2, 2013, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TN7cAAG&field=File__Body__s>.
```
Comment (jimfenton) [inline sp800-63a/sec10_references.md]: I think X.509 is part of the title and should be inside the asterisks. But in checking this I found that the URL is incorrect.

File: sp800-63a/sec10_references.md [hunk 2]
```diff
@@ -6,27 +6,25 @@
...
 
-<a name="fbcasup"></a>[FBCASUP] FBCA Supplementary Antecedent, In-Person Definition, July 16, 2009, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TNPgAAO&field=File__Body__s>.
+<a name="fbcasup"></a>[FBCASUP] *FBCA Supplementary Antecedent, In-Person Definition*, July 16, 2009, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TNPgAAO&field=File__Body__s>.
```
Comment (jimfenton) [inline sp800-63a/sec10_references.md]: This URL doesn't work either.

File: sp800-63a/sec10_references.md [hunk 3]
```diff
@@ -36,9 +34,30 @@
...
+NIST 800 Series Special Publications are available at: <http://csrc.nist.gov/publications/nistpubs/index.html>. The following publications may be of particular interest to those implementing systems of applications requiring e-authentication.
+
+<a name="SP800-53"></a>[SP 800-53] NIST Special Publication 800-53 Revision 4, *Recommended Security and Privacy Controls for Federal Information Systems and Organizations*, April 2013 (updated January 22, 2015), <https://doi.org/10.6028/NIST.SP.800-53r4>.
+
+<a name="SP800-63-3"></a>[SP 800-63-3] NIST Special Publication 800-63-3, *Digital Identity Guidelines*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63-3>.
+
+<a name="SP800-63B"></a>[SP 800-63B] NIST Special Publication 800-63B, *Digital Identity Guidelines: Authentication and Lifecycle Management*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63b>.
+
+<a name="SP800-63C"></a>[SP 800-63C] NIST Special Publication 800-63C, *Digital Identity Guidelines: Authentication and Lifecycle Management*, June 2017, <
+https://doi.org/10.6028/NIST.SP.800-63c>.
```
Comment (jimfenton) [inline sp800-63a/sec10_references.md]: extraneous line break

File: sp800-63a/sec10_references.md [hunk 4]
```diff
@@ -36,9 +34,30 @@
...
+
+
+<a name="SP800-63B"></a>[SP 800-63B] NIST Special Publication 800-63B, *Digital Identity Guidelines: Authentication and Lifecycle Management*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63b>.
 
-<a name="SP800-157"></a>[SP 800-157] NIST Special Publication 800-157, *Guidelines for Derived Personal Identity Verification (PIV) Credentials*, December 2014.
+<a name="SP800-63C"></a>[SP 800-63C] NIST Special Publication 800-63C, *Digtial Identity Guidelines: Assertions and Federation*, June 2017, <https://doi.org/10.6028/NIST.SP.800-63c>.
```
Comment (jimfenton) [inline sp800-63a/sec10_references.md]: Digtial --> Digital

**Review by jimfenton — CHANGES_REQUESTED** [review]
A few issues yet.

File: sp800-63a/sec10_references.md (hunk 2 above)
Comment (dannago) [reply sp800-63a/sec10_references.md]: Can't find an authoritative source

File: sp800-63a/sec10_references.md [hunk 5]
```diff
@@ -6,27 +6,25 @@
...
 
 <a name="dmf"></a>[DMF] National Technical Information Service, *Social Security Death Master File*, available at: <https://www.ssdmf.com/Library/InfoManage/Guide.asp?FolderID=1>.
 
 <a name="E-Gov"></a>[E-Gov] *E-Government Act of 2002* \(includes FISMA) (P.L. 107-347), December 2002, available at: <http://www.gpo.gov/fdsys/pkg/PLAW-107publ347/pdf/PLAW-107publ347.pdf>.
 
-<a name="9397"></a>[EO 9397] *Executive Order 9397, Numbering System for Federal Accounts Relating to Individual Persons*, November 22, 1943, available at: <https://www.ssa.gov/foia/html/EO9397.htm>.
-
-<a name="fbcacp"></a>[FBCACP] X.509 Certificate Policy For The Federal Bridge Certification Authority (FBCA), Version 2.27, December 2, 2013, available at: <https://www.idmanagement.gov/IDM/servlet/fileFieldentityId=ka0t0000000TN7cAAG&field=File__Body__s>.
+<a name="fbcacp"></a>[FBCACP] *X.509, Certificate Policy For The Federal Bridge Certification Authority (FBCA)*, Version 2.3, October 5, 2016, available at: <https://www.idmanagement.gov/wp-content/uploads/sites/1171/uploads/FBCA_CP.pdf>.
```
Comment (jimfenton) [inline sp800-63a/sec10_references.md]: No comma after X.509 and version number should be 2.30 (it's not a decimal number, so 2.3 is different)

**Review by jimfenton — APPROVED** [review]

**Pull request merged**